docs/contributor/arch/002-fluent-bit-without-file-system-buffer.md

```diff
@@ -0,0 +1,63 @@
+# 1. Fluent Bit without File-System Buffer
+
+Date: 2023-11-23
+
+## Status
+
+Proposed
+
+## Context
+
+The Fluent Bit configuration that we currently generate with the telemetry-manager from a LogPipeline uses a `tail`
+input plugin to read container logs, splits the log stream into multiple pipelines using a `rewrite_tag` filter, applies
+additional pipeline specific filters, and sends the log stream to the configured output. This setup decouples the input
+and outputs so that a blocked output does not affect all other outputs. Consequently, reading logs by the tail plugin is
+never paused. The `rewrite_tag` filter uses a persistent file-system buffer to prevent log loss. The persistent buffer
+is limited to 1 GB to ensure that Kubernetes nodes do not run out of disk space.
+
+The persistent buffer can ensure log loss only for a short period of time. Depending on the amount of generated logs,
```

Comment:
Please confirm: Does the buffer ENSURE log loss? I thought it prevents log loss.

```suggestion
The persistent buffer can ensure {prevent} log loss only for a short period of time. Depending on the amount of generated logs, this is usually in the range of minutes to a few hours and might be too short to restore an outage of a log backend or detect and solve a faulty configuration.
```

Reply:
Good catch. It prevents log loss.